protoc_plugin/lib/src/protobuf_field.dart

```diff
@@ -62,9 +62,11 @@ class ProtobufField {
   bool get isRepeated =>
       descriptor.label == FieldDescriptorProto_Label.LABEL_REPEATED;
 
-  /// True if a numeric field is repeated and it should be encoded with packed
-  /// encoding. In proto3 repeated fields are encoded as packed by default.
-  /// proto2 requires `[packed=true]` option.
+  /// Whether a numeric field is repeated and it must be encoded with packed
+  /// encoding.
+  ///
+  /// In proto3 repeated fields are encoded as packed by default. proto2
```

Comment:
Avoid starting a sentence with a lower-case letter.
Either "Proto2 requires" or "The proto2 format requires" .... (or whichever noun is appropriate, if "format" isn't).